streams/src/main/java/org/apache/kafka/streams/state/internals/LogicalKeyValueSegments.java

```diff
@@ -57,6 +57,11 @@ public void openExisting(final ProcessorContext context, final long streamTime)
         physicalStore.openDB(context.appConfigs(), context.stateDir());
     }
 
+    @Override
+    public void cleanupExpiredSegments(final long streamTime) {
+        super.cleanupExpiredSegments(streamTime);
```

Comment:
The method from AbstractSegments is protected, but LogicalKeyValueSegments needs to expose it publicly so that it can be called from the versioned store implementation.